nft/nfts.yaml

```diff
@@ -854,16 +826,24 @@ paths:
       tags: ['Sales & Marketplace Data']
       parameters:
         - $ref: '#/components/schemas/apiKey'
+          required: true
         - $ref: '#/components/schemas/fromBlock'
+          required: true
         - $ref: '#/components/schemas/toBlock'
+          required: true
         - $ref: '#/components/schemas/order'
+          required: true
         - $ref: '#/components/schemas/marketplace'
+          required: true
         - $ref: '#/components/schemas/contractAddress'
           description: String - The contract address of a NFT collection to filter sales by. Defaults to returning all NFT contracts.
+          required: true
```

Comment:
i don’t think these are all required

Reply:
but can u double check?

Reply:
I'll just revert with a new PR and cleared all conflicts.